src/components/layout/binary-to-deriv/footer.js

```diff
@@ -5,14 +5,13 @@ import { Container } from '../../containers'
 import { FooterGrid } from '../footer/common/style.js'
 import LogoSection from '../footer/logo'
 import DisclaimerSection from './_disclaimer'
-import CopyrightSection from './_copyright'
-import { Flex } from 'components/containers'
 import device from 'themes/device'
 
 const DefaultFooter = styled.footer`
     background-color: var(--color-grey-25);
     width: 100%;
     margin: 0 auto;
+    padding-bottom: 2rem;
```

Comment:
Added the padding here because of removing the copyright section.


<img width="1438" alt="Screen Shot 2022-01-20 at 3 35 10 PM" src="https://user-images.githubusercontent.com/64970259/150335756-df342051-b99a-40fb-b9d1-7128afa9e754.png">
<img width="1440" alt="Screen Shot 2022-01-20 at 3 35 21 PM" src="https://user-images.githubusercontent.com/64970259/150335814-41465894-d522-49ef-89fc-5366a932d3c0.png">